src/main/scala/diplomacy/BundleBridge.scala

```diff
@@ -42,16 +43,26 @@ case class BundleBridgeSink[T <: Data](genOpt: Option[() => T] = None)
 {
   def bundle: T = in(0)._1
 
-  def makeIO()(implicit valName: ValName): T = makeIOs()(valName).head
-  def makeIO(name: String): T = makeIOs()(ValName(name)).head
+  def makeIO()(implicit valName: ValName): T = {
+    val io: T = IO(chiselTypeClone(bundle))
+    io.suggestName(valName.name)
+    io <> bundle
+    io
+  }
+  def makeIO(name: String): T = makeIO()(ValName(name))
 }
 
 case class BundleBridgeSource[T <: Data](genOpt: Option[() => T] = None)(implicit valName: ValName) extends SourceNode(new BundleBridgeImp[T])(Seq(BundleBridgeParams(genOpt)))
 {
   def bundle: T = out(0)._1
 
-  def makeIO()(implicit valName: ValName): T = makeIOs()(valName).head
-  def makeIO(name: String): T = makeIOs()(ValName(name)).head
```

Comment:
@jackkoenig before this PR, using the HeterogenousBag IO maker built in to the generic diplomacy `makeIOs` i ended up with `reset_vector_N_0` 😂 

@mwachs5 This is a case where we could potentially use RecordMap to have the reset vectors be named for the id of the tile they are attached to rather than just the index of the IO (and hoping those coincidentally match up if all the static ids are contiguous)

Reply:
ok hoping to clean up the PRCI on Record Map PR so that RecordMap could go in first...

Reply:
Oh yeah, i just mention it as more of a "possible future work that could maybe also use the same feature." Since it changes a subsystem IO name I think we would have to figure out how to "productize" the change...